Elevel/Application/Features/TopicCommands/UpdateTopicCommand.cs

```diff
@@ -13,9 +11,12 @@ namespace Elevel.Application.Features.TopicCommands
 {
     public class UpdateTopicCommand
     {
-        public class Request : UpsertTopicCommand.Request, IRequest<Response>
+        public class Request : IRequest<Response>
         {
             public Guid Id { get; set; }
+            public string TopicName { get; set; }
+            public Level Level { get; set; }
+            public DateTimeOffset CreationDate { get; set; }
```

Comment:
CreationDate Set by ORM in configuration files

Reply:
Deleted